src/console.h

```diff
@@ -3,7 +3,7 @@
 
 #include "platform.h"
 
-#define CONSOLE_BUFFER_SIZE 2048
+#define CONSOLE_BUFFER_SIZE 3072
```

Comment:
Config listing now takes longer than 2048 chars. TODO we need a better way to handle this.